grafana-plugin/src/components/ScheduleQualityDetails/ScheduleQualityDetails.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+
+import { HorizontalGroup, Icon, IconButton } from '@grafana/ui';
+import cn from 'classnames/bind';
+import dayjs from 'dayjs';
+
+import Text from 'components/Text/Text';
+import { ScheduleScoreQualityResponse, ScheduleScoreQualityResult } from 'models/schedule/schedule.types';
+import { getTzOffsetString } from 'models/timezone/timezone.helpers';
+import { User } from 'models/user/user.types';
+import { useStore } from 'state/useStore';
+import { getVar } from 'utils/DOM';
+
+import styles from './ScheduleQualityDetails.module.scss';
+import { ScheduleQualityProgressBar } from './ScheduleQualityProgressBar';
+
+const cx = cn.bind(styles);
+
+interface ScheduleQualityDetailsProps {
+  quality: ScheduleScoreQualityResponse;
+  getScheduleQualityString: (score: number) => ScheduleScoreQualityResult;
+}
+
+export const ScheduleQualityDetails: FC<ScheduleQualityDetailsProps> = ({ quality, getScheduleQualityString }) => {
+  const { userStore } = useStore();
+  const { total_score: score, comments, overloaded_users } = quality;
+  const [expanded, setExpanded] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [overloadedUsers, setOverloadedUsers] = useState<User[]>([]);
+
+  useEffect(() => {
+    fetchUsers();
+  }, []);
+
+  const handleExpandClick = useCallback(() => {
+    setExpanded((expanded) => !expanded);
+  }, []);
+
+  if (isLoading) {
+    return null;
+  }
+
+  const infoComments = comments.filter((c) => c.type === 'info');
```

Comment:
`useMemo` here and below looks more `reactish` but doesn't make sense a lot

Reply:
`useMemo` seems an overkill, it's a trivial JSON data, a few comments at most.